mteb/abstasks/languages.py

```diff
@@ -2,7 +2,10 @@
 Script codes (ISO 15924) obtained from: https://unicode.org/iso15924/iso15924.txt
 """
 
+from __future__ import annotations
+
 import json
+from dataclasses import dataclass
 from pathlib import Path
```

Comment:
```suggestion
from dataclasses import dataclass
import json
from pathlib import Path
```

Reply:
I am not sure I get this? Imports are auto-sorted by ruff

Reply:
Oh sorry my bad!